docs/components/modeler/desktop-modeler/element-templates/defining-templates.md

```diff
@@ -242,6 +242,38 @@ Note that the configuration options `editable` and `constraints` will have no ef
 For the `property`, `camunda:property`, `camunda:in`, `camunda:in:businessKey`, `camunda:out` and `camunda:field` bindings, an omitted `type` will lead to rendering the `String` component (single line input).
 
 For the `camunda:executionListener` binding, an omitted `type` will lead to the `Hidden` component (ie. no visible input for the user).
+
+###### Feel
+
+As of Camunda Modeler `v5.0.0`, we support the feel properties `optional` and `required`.
+When set, the input field offers a visual indications that a feel expression is expected.
+
+```json
+  "properties": [
+    {
+      "label": "Optional Feel Expression",
+      "type": "String",
+      "feel": "optional"
+    },
+    {
+      "label": "Required Feel Expression",
+      "type": "Text",
+      "feel": "required"
+    }
+  ]
+```
+
+__Supported Types__
```

Comment:
With using `###### Feel` above, this heading now looks bolder

![image](https://user-images.githubusercontent.com/9433996/159259157-aa81fed0-aac9-4254-bd5d-2e13d954af0a.png)

Can we maybe use `####### Supported Types`?

Reply:
Unfortunately, the HTML standard only defines 6 Heading, so it would not work.

We skip 1 heading level between Types and the Details, so we can give Feel H5 and `Supported Types` H6

Reply:
h5 and h6 are still the same size, but at least it is not bigger...

![image](https://user-images.githubusercontent.com/21984219/159261239-be206682-9f39-4daa-9304-0af938a214c4.png)

Reply:
It even is a smaller-ish font size (0.85 em vs 0.875 em) :D